app/views/catalog/_document_recent_items.html.erb

```diff
@@ -0,0 +1,4 @@
+<% # container for all documents in index view -%>
```

Comment:
are these names required by the solr document helper? It seems like it would be nice to have them flipped around aka `_recent_items_document` and `_recent_items_index`

Reply:
Also why can't they live in `app/views/spotlight/sir_trevor/blocks/` with the other template?